inline_scan.sh

```diff
@@ -165,7 +173,7 @@ get_and_validate_analyzer_options() {
         printf '\n\t%s\n\n' "ERROR - must specify a valid sha256:<digestID>: ${SYSDIG_IMAGE_DIGEST}" >&2
         display_usage_analyzer >&2
         exit 1
-    elif ! curl -k -s --fail -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_ANCHORE_URL}/images" > /dev/null; then
+    elif ! curl -s "${CURL_AUTH}" --fail -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_ANCHORE_URL}/images" > /dev/null; then
```

Comment:
I don't understand the point on adding --basic (if -l option is passed) to curl, because we don't specify the user -u and we are already adding an "Authorization: Bearer ..." header, right? I guess the --basic option would be used to use "Authorization: Basic ..." instead?